app/helpers/editorial_helper.rb

```diff
@@ -1,7 +1,9 @@
 module EditorialHelper
 
   def form_type(node_clazz)
-    "#{node_clazz.name}Form".constantize
+    # Because of development mode autoloading, the class may not exist until
+    # we call constantize. Hence just catching the exception.
+    "#{node_clazz.name}Form".constantize rescue NodeForm
```

Comment:
We can create the subclass. I just don't want to because there's nothing to put in it and I can see this happening again in the future. I can specifically rescue from the correct exception if that makes it any better. I also didn't actually test that the is-this-constant-defined worked or not in dev. I'll try that too.